xapian-bindings/configure.ac

```diff
@@ -94,6 +94,24 @@ XO_LIB_XAPIAN([], [],
 			  [^\([0-9]*\.[0-9]*[13579]\)\..*$], [-\1]))
 XO_REQUIRE([1.4.0])
 
+AC_ARG_ENABLE([letor],
+  [AS_HELP_STRING([--enable-letor], [])],
+  [case ${enableval} in
+    yes) ;;
+    no)
+      AC_MSG_WARN([xapian-letor is unavailable, hence letor-bindings won't be built])
```

Comment:
This warning doesn't really make sense.  We'll only emit it if the user explicitly ran something like `./configure --disable-letor` and if they did that it shouldn't be a surprise that letor was disabled.  Also, the warning message is wrong - we're disabling it because we were told to - we haven't checked if it was available or not at this point.

Reply:
fixed in a fixup commit.